src/creature_states_combt.c

```diff
@@ -1791,16 +1791,6 @@ long ranged_combat_move(struct Thing *thing, struct Thing *enmtng, MapCoordDelta
         return -inst_id; \
     }
 
-TbBool creature_would_benefit_from_healing(const struct Thing* thing)
```

Comment:
You say this function is a duplicate of creature_requires_healing, and that you therefore removed it, but it most certainly was not a duplicate and I would love to get it back.

One function looks at crstat->heal_requirement, the other at crstat->heal_threshold.
The first one checks if the health is so low that healing is an absolute priority. The other one checks if you have enough damage to be willing to spend a heal-spell on it if you have nothing else to do.